### ISSUE TITLE
Switching to IFrame throw TypeError: Cannot call a class as a function.

### ISSUE THREAD

**Issue by ljowen** [DevExpress/testcafe#3115]
### What is your Test Scenario?
Test contents of Facebook Video embedded via Iframe
 
### What is the Current behavior?
Testcafe throws an error: 
```    
1) An error occurred in Selector code:

           TypeError: Cannot call a class as a function
 ```
### What is your web application and your TestCafe test code?
 
```js
 import { t, Selector } from 'testcafe';

fixture('FB Video').page(
  'http://developers.facebook.com/docs/plugins/embedded-video-player/'
);

test('example', async () => {
  const fbVideoIframe = Selector(
    'iframe[src*="facebook.com"][title="fb:video Facebook Social Plugin"]'
  );

  await t.hover('h2#how-to-get-a-video-posts-url');

  await t.switchToIframe(fbVideoIframe);
});

```

### Steps to Reproduce:

1. Go to my website ...
3. Execute this command...
4. See the error...
 
### Your Environment details:
 
* testcafe version: 0.23.2 / 0.22.0                   
* node.js version: 8.12                   
* command-line arguments:  testcafe chrome,firefox,safari  --skip-js-errors 
* browser name and version: firefox63, chrome70, safari11
* platform and version: MacOS 10.13
* other

**Comment by ljowen** [comment]
Note this a derived example. 
Interestingly on my site (which contains facebook video embeds) I only see the error on mobile browsers, the tests pass fine for desktop.

**Comment by AndreyBelym** [comment]
I've reproduced the error you described but also encountered another problem that can cause the original issue. You can see its details in a separate thread: DevExpress/testcafe-hammerhead#1834. We need to fix it before we can address the original problem.

**Comment by gcofficial** [comment]
@AndreyBelym
Hi guys. I have the same issue im my project. When do you plan to fix this?

**Comment by dobharweim** [comment]
Any workarounds on this? I cam getting the same error when I try to access iFrame by position - but only certain pages.

`        .switchToIframe(Selector(() => {
        var lst =  document.querySelectorAll("*[id^='iframe']");
        return lst[1];
        }))`

**Comment by devmondo** [comment]
same happening here, we got iframe with random id coming up every time. we need work around please

**Comment by miherlosev** [comment]
It's a duplicate of #3681.
This problem is fixed in `testcafe@1.1.3`.
